### DISCUSSION TITLE
Introducing UmiJS for more developers

### DISCUSSION THREAD

**Discussion by Douglas0n** [umijs/umi#7567] [Q&A]
We have a lot of books in English about Next.js, but not that much about UmiJS, with that in mind, I'm working on a project to introduce UmiJS to entry-level developers.

I'm looking for insights from the community on addressing developer needs in using UmiJS for front-end development.

* What topics do you feel you need help with to deepen your knowledge about UmiJS?
* Are you interested in understanding key concepts to develop front-end applications like SPAs and PWAs?
* Do you have any other suggestions?

I'm excited about this project and open to your opinion.

**Comment by akmalelias** [comment]
Hi, I'm wondering if there is any intention to translate the UmiJS website and documentation into English?

**Reply by Douglas0n** [reply]
Hi, @akmalelias, Actually its one of the items in 2023 Roadmap: #10423 - （文档）Umi 中文到英文翻译脚本，diff 精确到行 @sorrycc
(Documentation - Chinese to English script, accurate line diff).

By the way, I have published the book in collaboration with Packt covering UmiJS v3, if you are interested: https://amzn.to/3KVYmxh

**Reply by sobhardwaj** [reply]
at least the roadmap also in English as well ..
